draft-ietf-httpbis-client-cert-field.md

```diff
@@ -179,15 +194,13 @@ NOT be used in HTTP responses.  It MAY have a list of values or occur multiple
 times in a request.  For header compression purposes, it might be advantageous
 to split lists into multiple instances.
 
-The first certificate in the list SHOULD directly certify the end-entity
+The first certificate in the list directly certifies the end-entity
 certificate provided in the `Client-Cert` header; each following certificate
-SHOULD directly certify the one immediately preceding it.  Because certificate
+directly certifies the one immediately preceding it.  Because certificate
```

Comment:
Replied in the now-obsolete thread, but the SHOULDs come from RFC 8446's version of these requirements.  If we're making this mandatory (implicit MUST), this needs to specify what happens when it's violated.

SHOULD means exactly that -- it's better to do it this way, and things might go wrong if you don't.

Reply:
My hope/goal was to place constraints on the sender here that lead to a predictable output. Do it this way. Full stop. 

I'm not sure what happens when it's violated with must or should. I see what you are saying but the implicit MUSTs are meat to mean - do it this way, otherwise it's wrong.